pwa-devdocs/src/tutorials/intercept-a-target/change-static-assets-directory/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Change static assets directory
+adobeio: /tutorials/targets/change-static-assets-directory/
```

Comment:
Need to port this content over to the new site

Reply:
Done